node-console.js

```diff
@@ -46,3 +46,35 @@ OrderContext = _orders.OrderContext;
 User = (r("models/users")).User;
 AuthToken = (r("models/users")).AuthToken;
 Application = (r("models/applications")).Application;
+AppOffer = r('models/applications').AppOffer;
```

Comment:
why is this file here and not (in ts) with the rest of them scripts?

Reply:
This needs to be ready even before you build the files.

Reply:
why?

Reply:
mmm...I had a big written explanation about when this needs to be ready but it doesn't matter cause it has TS dependencies...I'll convert it

Reply:
Since this file wasn't added in this PR but only had some minor changes & this PR has been waiting for a long time, I'll convert this in a dedicated PR.